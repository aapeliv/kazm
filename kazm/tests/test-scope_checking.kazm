int main() {
  int i = 0;
  while (i < 10) {
    i = i + 1;
    int_print(i);
  }
  println("");
  { int j = 9;
    int_print(j+1);
    { int m = 9;
      j = j + 1;
      int_print(j);
    }
  }
  println("");
  {
    string j = "hello";
    println(j);
  }
  string j = "world";
  while(i < 11) {
    i = i + 1;
    println(j);
  }
}
