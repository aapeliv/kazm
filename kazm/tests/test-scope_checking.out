12345678910
1010
hello
world
